localflavor/generic/forms.py

```diff
@@ -127,3 +130,13 @@ def prepare_value(self, value):
         if value is not None:
             return value.upper()
         return value
+
+
+class PhoneNumberFormFieldMixin(object):
+    def __init__(self):
+        super(PhoneNumberFormFieldMixin, self).__init__()
+        warnings.warn(
```

Comment:
@timgraham I agree this is a good way to go. I'll update the docs that asks for u prefixed strings.
https://django-localflavor.readthedocs.io/en/latest/#adding-flavors